### ISSUE TITLE
Untranslatable output "Is this ok"

### ISSUE THREAD

**Issue by rffontenelle** [rpm-software-management/dnf5#1927]
The following message is not extracted by gettext, hence is not available for translation:

https://github.com/rpm-software-management/dnf5/blob/f1f8723bd31de6212e0e6611e0f753e0c36aabdc/include/libdnf5-cli/utils/userconfirm.hpp#L41-L48

**Comment by ppisar** [comment]
It's partially on purpose because the prompts does not yet support localized answers (Y/N). And that requires setting C++ locale [https://github.com/rpm-software-management/dnf5/pull/1699] which is not yet properly implemented.

I could mark it for translation with a note that Y/N should not be translated. But I worry the translators will ignore it and that will confuse the users.

**Comment by rffontenelle** [comment]
@ppisar How about something like a translation string "Is this ok %s" where the placeholder %s is replaced with [Y/n] and [y/N] ?  This way translators won't be able to mess the y/n

**Comment by ppisar** [comment]
I was also thinking at it (proper internationalization would be "Is this ok [%s/%s]: "), but now I looked at the source code and more serious problem is that the code is part of a C++ template. That is it's compile-time thing. The confirmation prompt first needs to be move to a library before we can internationalize it.

**Comment by sunwire** [comment]
Hey guys, I reported this issue earlier but it seems to have gone unnoticed. https://github.com/rpm-software-management/dnf5/issues/1868

**Comment by ppisar** [comment]
@sunwire, thanks for hint. I was on vacation and still not yet read all e-mails.

Closing this ticket as duplicate of #1868.
